beitragsordnung.md

```diff
@@ -2,13 +2,16 @@
 
 Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 folgende Beitragsordnung beschlossen:
 
-1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
+1. Miedergebühren
+1.1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
 
-2. Ehrenmitglieder sind von der Beitragszahlung befreit. In Sonderfällen können gemäß Satzung § 6 auch von ordentlichen Mitgliedern mit Stimmrecht begründete Anträge auf Ermäßigung der Beitragshöhe gestellt werden.
+1.2. Ehrenmitglieder sind von der Beitragszahlung befreit. 
```

Comment:
Wer und wie kann man Ehrenmitglied werden? Davon steht nichts in der Satzung

Reply:
Das ist sicher ein guter Punkt. Wäre in der Satzung aufzunehmen, oder? Vllt. können wir den Punkt in der Beitragsordnung erstmal rausnehmen.

Reply:
Ja, das müsste man dann erst in der Satzung klären. Ich wäre daher auch für aus der Beitragsordnung rausnehmen.

Reply:
Dann nimm es doch gerne direkt raus.

Reply:
Ist raus.